packages/e2e-tests/specs/editor/various/invalid-block.test.js

```diff
@@ -32,8 +32,6 @@ describe( 'invalid blocks', () => {
 
 		// Takes the focus away from the block so the invalid warning is triggered
 		await page.click( '.editor-post-save-draft' );
-		expect( console ).toHaveErrored();
```

Comment:
I tried the same test manually in the browser. It works differently.

Before this PR:

<img width="807" alt="Screen Shot 2021-08-09 at 13 06 40" src="https://user-images.githubusercontent.com/699132/128697743-3d885be7-6192-4349-96f8-1d8b03fd4eac.png">

With this PR:

<img width="1133" alt="Screen Shot 2021-08-09 at 13 10 51" src="https://user-images.githubusercontent.com/699132/128697788-ef740a5e-2a3b-4af3-be86-f2310f07e17f.png">

Reply:
We can always trigger the logs if we want but for me the `isValidBlockContent` function should not log errors automatically, it's a function that returns a boolean, it's the responsibility of the caller to decide whether to log errors or not.

for the this specific test, I didn't think that the logs are necessary since there's a UI feedback showing the diff... that said, if you think otherwise, we can always bring the logs back in this flow.

Reply:
When I clock resolve I see the following:

<img width="860" alt="Screen Shot 2021-08-09 at 13 16 14" src="https://user-images.githubusercontent.com/699132/128698286-5621d072-8311-4aa2-80f2-bd39f68ff975.png">

As discussed with @youknowriad. Everything is handled in the UI so there is no need for logging into the console. It probably bails out early because it detects the content as the freeform block.